### PR TITLE
[ESS & Serverless] Adds OrganizationUnitIDs description to agentless section of AWS CSPM guide

### LINKED ISSUE #5636: [enhancement] CSPM onboarding "Organization
### Description

Per [this slack thread](https://elastic.slack.com/archives/C6E3MTCD7/p1721229945094309?thread_ts=1721201411.526719&cid=C6E3MTCD7), we should document the purpose of the OrganizationUnitIDS field while setting up CSPM. You can use it to specify which of the accounts within an Organization to scan.

### Background & resources

* PRs: 
* Issues/metas: 
* Point of contact: Eyal, Smriti
* Test environments: 

### Which documentation set does this change impact?

ESS and serverless

### ESS release

8.14

### Serverless release

-

### Feature differences

-

### API docs impact

-

### Prerequisites, privileges, feature flags

_No response_

### PR DESCRIPTION
**Pull request by benironside**
Fixes #5636 
Adds OrganizationalUnitIDs field to the agentless CSPM AWS guides, edits the field's description in agent-based method.

Preview: CSPM for AWS ([ESS](https://security-docs_bk_6155.docs-preview.app.elstc.co/guide/en/security/master/cspm-get-started.html)), ([Serverless](https://security-docs_bk_6155.docs-preview.app.elstc.co/guide/en/serverless/current/security-cspm-get-started.html))

### CODE REVIEW

File: docs/cloud-native-security/cspm-get-started-aws.asciidoc [hunk 1]
```diff
@@ -76,7 +79,7 @@ For most use cases, the simplest option is to use AWS CloudFormation to automati
 . Return to your {kib} tab. Click *Save and continue* at the bottom of the page.
 . Review the information, then click *Launch CloudFormation*.
 . A CloudFormation template appears in a new browser tab.
-. For organization-level deployments only, you must enter the ID of the organizational unit where you want to deploy into the `OrganizationalUnitIds` field in the CloudFormation template. You can find it in the AWS console under *AWS Organizations -> AWS Accounts* (it appears under the organization name).
+. For organization-level deployments only, you must enter the ID of the organizational unit(s) where you want to deploy into the CloudFormation template's `OrganizationalUnitIds` field. You can find organizational unit IDs in the AWS console under *AWS Organizations -> AWS Accounts* (under each organization's name). You can also use this field to specify which accounts in your organization to monitor, and which to skip.
```
Comment (nastasha-solomon) [inline docs/cloud-native-security/cspm-get-started-aws.asciidoc:82]: I don't think we have formal guidance on this, but I'm fairly certain we don't use the parentheses `s` to indicate that a noun can be singular or plural. I've always followed the rule of choosing the best option for the scenario described in the docs.

```suggestion
. For organization-level deployments only, you must enter the ID of the organizational units where you want to deploy into the CloudFormation template's `OrganizationalUnitIds` field. You can find organizational unit IDs in the AWS console under *AWS Organizations -> AWS Accounts* (under each organization's name). You can also use this field to specify which accounts in your organization to monitor, and which to skip.
```

File: docs/serverless/cloud-native-security/cspm-get-started.asciidoc [hunk 2]
```diff
@@ -79,7 +82,7 @@ For most use cases, the simplest option is to use AWS CloudFormation to automati
 . Return to your {kib} tab. Click **Save and continue** at the bottom of the page.
 . Review the information, then click **Launch CloudFormation**.
 . A CloudFormation template appears in a new browser tab.
-. For organization-level deployments only, you must enter the ID of the organizational unit where you want to deploy into the `OrganizationalUnitIds` field in the CloudFormation template. You can find it in the AWS console under **AWS Organizations → AWS Accounts** (it appears under the organization name).
+. For organization-level deployments only, you must enter the ID of the organizational unit(s) where you want to deploy into the CloudFormation template's `OrganizationalUnitIds` field. You can find organizational unit IDs in the AWS console under *AWS Organizations -> AWS Accounts* (under each organization's name). You can also use this field to specify which accounts in your organization to monitor, and which to skip.
```
Comment (nastasha-solomon) [inline docs/serverless/cloud-native-security/cspm-get-started.asciidoc:85]:
```suggestion
. For organization-level deployments only, you must enter the ID of the organizational units where you want to deploy into the CloudFormation template's `OrganizationalUnitIds` field. You can find organizational unit IDs in the AWS console under *AWS Organizations -> AWS Accounts* (under each organization's name). You can also use this field to specify which accounts in your organization to monitor, and which to skip.
```

**Review by nastasha-solomon — COMMENTED** [review]
Just two small edits!

**Review by smriti0321 — DISMISSED** [review]
LGTM

**Pull request merged**